src/type_checker/resources/std_lib/collection.py

```diff
@@ -0,0 +1,21 @@
+from typing import TypeVar, Generic
+
+class set(Generic[T]):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'set'](https://app.codacy.com/app/JSAbrahams/mamba/pullRequest?prid=4163821)